src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+/* styles for big search-box */
+
+.input {
```

Comment:
where is font-weigh style? it should be 300 by default, 400 for placeholder and 900 on focus